plugins/network_broadcast_api/network_broadcast_api.cpp

```diff
@@ -39,12 +42,12 @@ namespace golos {
             }
 
             DEFINE_API(network_broadcast_api_plugin, broadcast_transaction) {
-                auto n_args = args.args->size();
-                FC_ASSERT(n_args == 1, "Expected at least 1 argument, got 0");
-                auto trx = args.args->at(0).as<signed_transaction>();
-                if (n_args > 1) {
-                    const auto max_block_age = args.args->at(1).as<uint32_t>();
-                    FC_ASSERT(!check_max_block_age(max_block_age));
+                PLUGIN_API_VALIDATE_ARGS(
+                    (signed_transaction, trx)
+                    (uint32_t,           max_block_age, 0)
+                );
+                if (args.args->size() >= 2) {
```

Comment:
`PLUGIN_API_VALIDATE_ARGS` defines `auto n_args = args.args->size();`, so `n_args` can be used here

Reply:
Fixed